### PR TITLE
Add permissions skipping feature behing flag

### PR DESCRIPTION
**Pull request by CPatchane**
Allow to skip permissions when installing app or konnectors when the label is below a configurable `authorizedLabelLimit` number

### CODE REVIEW

**Comment by CPatchane** [comment]
[(𝐌) [STO] ETUO je n'ai pas de permissions dans mon parcours de configuration d'un connecteur bancaire](https://trello.com/c/UJmtHcRo/1128-%F0%9D%90%8C-sto-etuo-je-nai-pas-de-permissions-dans-mon-parcours-de-configuration-dun-connecteur-bancaire)

File: src/ducks/apps/components/AppInstallation.jsx [hunk 1]
```diff
@@ -16,16 +17,27 @@ import Checkbox from 'cozy-ui/react/Checkbox'
...
+
 export class AppInstallation extends Component {
   constructor(props) {
     super(props)
+    const { app } = props
     this.state = { isTermsAccepted: false }
+    if (shouldSkipPermissions(app) && hasInstallation(app)) {
+      this.installApp()
+    }
+    this.installApp = this.installApp.bind(this)
```
Comment (gregorylegarec) [inline src/ducks/apps/components/AppInstallation.jsx]: I would have maybe binded this method before calling `this.installApp()` right above. It feels strange to call it and then bind it.

File: test/ducks/apps/__snapshots__/index.spec.js.snap [hunk 2]
```diff
@@ -28,6 +28,7 @@ Array [
         "icon": "<svg></svg>",
         "installed": false,
         "isInRegistry": true,
+        "label": 2,
```
Comment (gregorylegarec) [inline test/ducks/apps/__snapshots__/index.spec.js.snap]: It's probably too late but I find the word `label` too generic.

File: src/ducks/components/App.jsx [hunk 3]
```diff
@@ -43,12 +43,12 @@ export class App extends Component {
   }
 }
 
-const mapStateToProps = state => ({
+export const mapStateToProps = state => ({
```
Comment (gregorylegarec) [inline src/ducks/components/App.jsx]: Why not isolate them into a module ? It feels more natural to me to override a module but I could be missing a point.

File: src/ducks/apps/components/AppInstallation.jsx [hunk 4]
```diff
@@ -16,16 +17,27 @@ import Checkbox from 'cozy-ui/react/Checkbox'
 
 import { getTranslatedManifestProperty } from 'lib/helpers'
 import { hasPendingUpdate } from 'ducks/apps/appStatus'
+import CTS from 'config/constants'
```
[the remark this answers is not in the document]
Comment (CPatchane) [reply src/ducks/apps/components/AppInstallation.jsx]: It was just an abbreviation for constants to be less long

**Pull request merged**